### PR TITLE
Update rebar3 docs link

### PR DESCRIPTION
**Pull request by ferd**
3.14 moved the docsite around and this one is outdated.

### CODE REVIEW

File: lib/hexpm_web/templates/docs/mirrors.html.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ To permanently select a mirror, simply run the following command in a shell sess
 
 **Mix Example**: `$ mix hex.config mirror_url https://repo.hex.pm`
 
-**Rebar3 Example**: Add to the global or a project's top level `rebar.config`, `{rebar_packages_cdn, "https://repo.hex.pm"}.`. For more information see rebar3's package support and configuration [documentation](http://www.rebar3.org/docs/dependencies#package-manager-dependencies).
+**Rebar3 Example**: Add to the global or a project's top level `rebar.config`, `{rebar_packages_cdn, "https://repo.hex.pm"}.`. For more information see rebar3's package support and configuration [documentation](https://rebar3.org/docs/dependencies/).
```
Comment (ericmj) [inline lib/hexpm_web/templates/docs/mirrors.html.md:13]: The old anchor link does not seem to exist anymore so I just linked to the root instead, is that okay?

**Comment by ericmj** [comment]
I added a commit to your PR that updates the other places we link to rebar3.org.

**Review by ericmj — APPROVED** [review]

File: lib/hexpm_web/templates/docs/mirrors.html.md (hunk 1 above)
Comment (ferd) [reply lib/hexpm_web/templates/docs/mirrors.html.md:13]: yep

**Comment by ferd** [comment]
works for me

**Comment by ericmj** [comment]
Thank you

**Pull request merged**